examples/console/GB5.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
❗ Looks like this file was committed accidentally.

Reply:
yup removed
https://github.com/stellar/experimental-payment-channels/pull/311/commits/89aff901aa1a0a91aa7aef5f043ea85c3a7ff9c2